ethers-signers/src/wallet.rs

```diff
@@ -77,7 +77,7 @@ impl Wallet {
     // TODO: Add support for mnemonic and encrypted JSON
 
     /// Creates a new random keypair seeded with the provided RNG
-    pub fn new<R: Rng>(rng: &mut R) -> Self {
+    pub fn new<R: Rng + CryptoRng>(rng: &mut R) -> Self {
```

Comment:
The way we usually write these in RustCrypto is:

```suggestion
    pub fn new(mut rng: impl Rng + CryptoRng) -> Self {
```

...since at least RngCore has an impl of itself for its own reference types. I would assume `Rng` does too?